scripts/error_codes.py

```diff
@@ -202,7 +202,7 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
         "4591", # "There are more than 256 warnings. Ignoring the rest."
                 # Due to 3805, the warning lists look different for different compiler builds.
         "1834", # Unimplemented feature error, as we do not test it anymore via cmdLineTests
-        "5430", # basefee being used in inline assembly for EVMVersion < london
```

Comment:
Because of [this change](https://github.com/ethereum/solidity/pull/14755/files#diff-77a7a77f2a2831156ab9f1652dabc5af0858e8ff6739dd3890540ff1d3c5f875). Initially (in the original PR), it was like this, but the inline assembly part was then removed that would yield this error (5430). We've now decided to bring it back, but hardcode it to `berlin`, since the error changes depending on the EVM fork you're running the tests againsts.

Reply:
Ah ok, thanks for clarifying it;)